### PR TITLE
Issue 1981: Setting guidelines for contributors

### LINKED ISSUE #1981: Suggestions and clarifications around the CONTRIBUTING file
### What article on docs.github.com is affected?

- [Setting guidelines for repository contributors](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/setting-guidelines-for-repository-contributors)
- Source: https://github.com/github/docs/blob/main/content/github/building-a-strong-community/setting-guidelines-for-repository-contributors.md

### What part(s) of the article would you like to see updated?

A new section: **How GitHub uses the _CONTRIBUTING_ file**

- It appears as checked on the **Community** page
- It appears as the designated `CONTRIBUTING.*` guide on the repo `contribute` page
  - Example: https://github.com/github/docs/contribute
  - Side note: Is there actually documentation about how the `contribute` page works and is generated? Seems like an easter egg, sort of

Then there is this section:

> Decide whether to store your contributing guidelines in your repository's root, `docs`, or `.github` directory. Then, in the filename field, type the name and extension for the file. Contributing guidelines filenames are not case sensitive, and can have an extension such as `.md` or `.txt`. 

The _such as_ piece throws me off. Does it mean _any_ extension (such as `.rst`, often found in Python projects), or is there a limited like to the extensions accepted for the contributing file, or? This should be reworded for clarification. I have noticed that `.rst` does get picked up, so I know it is at least included.

The other problem with that section:
- For the `contribute` page, what `CONTRIBUTING.*` file path takes most importance? As in, what if a muddled project has the following:
  - `.github/CONTRIBUTING.md`
  - `docs/CONTRIBUTING.md`
  - `CONTRIBUTING.md`

What's the order of importance?

___
## Maintainer note:
Please follow the [Content design plan](https://github.com/github/docs/issues/1981#issuecomment-763535459) below to resolve this issue 🎉

### PR DESCRIPTION
**Pull request by muralishetty2558**
### Why:
Existing issue: https://github.com/github/docs/issues/1981

**Closes [issue link]**

### What's being changed: 
Updated md files
1 -  content/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors.md
2 - content/github/managing-your-work-on-github/managing-labels.md

### Check off the following:
- [ Not able to see deploy to heroku ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ **YES** ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ **YES** ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (eg. a "before and after video")

### CODE REVIEW

**Comment by janiceilene** [comment]
@muralishetty2558 Thanks so much for opening a PR with this fix! I'll triage it for review ⚡ 

You can see the Heroku deployment with your staging at https://docs-6110--issue-1981.herokuapp.com/en. I got there by clicking **View deployment** above ☝️ It can take a few minutes to show up after you've opened a PR.

image

File: content/github/managing-your-work-on-github/managing-labels.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ Label | Description
 
 Default labels are included in every new repository when the repository is created, but you can edit or delete the labels later.
 
+Issues with the `good first issue` label are used to populate the repository's contribute page.
```
Comment (saritai) [inline content/github/managing-your-work-on-github/managing-labels.md:52]:
```suggestion
Issues with the `good first issue` label are used to populate the repository's `contribute` page. For an example of a `contribute` page, see [github/docs/contribute](https://github.com/github/docs/contribute)
```

**Review by saritai — DISMISSED** [review]
Thanks for making these changes! I made a few minor wording updates and added a link, but otherwise it looks great to me. ✨  I will commit my minor updates, review in staging one more time, and merge shortly!

File: content/github/managing-your-work-on-github/managing-labels.md (hunk 1 above)
Comment (saritai) [reply content/github/managing-your-work-on-github/managing-labels.md:52]: Just added a link to an example contribute page, as mentioned in the content design plan!

**Review by saritai — APPROVED** [review]

**Pull request merged**